### PR TITLE
Button switch languages

### PR DESCRIPTION
**Pull request by vahona**
Hello there,

Finished the switch language feature, please have a look at it whenever you have time.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "react-native-dev-menu": "^4.0.2",
     "react-native-elements": "^3.4.0",
     "react-native-gesture-handler": "^1.10.3",
+    "react-native-i18n": "^2.0.15",
```
Comment (andrzej-onja) [inline package.json]: this package seems to be unused, remove please

Comment (andrzej-onja) [inline package.json]: this package seems to be unused, remove please

File: src/screens/Learning.js [hunk 2]
```diff
@@ -132,31 +127,31 @@ export default ({
             />
           </View>
           <View style={styles.heading}>
-            <SectionHeading text="Category: " />
+            <SectionHeading text={nativeLanguage === LANGUAGE_NAMES.EN ? "Category: " : "Sokajy :"} />
```
Comment (andrzej-onja) [inline src/screens/Learning.js]: create a file src/translations/index.js

```
export const CATEGORY_HEADING = 'CATEGORY_HEADING';


export const LANG_DATA = {
    [CATEGORY_HEADING]: {
    [LANGUAGE_NAMES.EN]: 'Category:',
    [LANGUAGE_NAMES.MG]: 'Sokajy: ',
  },
  
};
```

then you can get the translation like this

`const categoryHeading = LANG_DATA[CATEGORY_HEADING][nativeLanguage];`

then 
 ` <SectionHeading text={categoryHeading} />`

**Review by andrzej-onja — CHANGES_REQUESTED** [review]
So far looks good but we need another change:

please create file src/translations/index.js

File: src/screens/Home.js [hunk 3]
```diff
@@ -13,16 +13,27 @@ import {
...
+
+import { LANG_DATA } from '../translations';
+import {
+  CATEGORY_HEADING,
+  CATEGORY_LIST,
+  CATEGORY_SEEN_PHRASES_HEADING,
+  CATEGORY_SEEN_PHRASES,
+  CATEGORY_LEARNT_PRASES_HEADING,
+  CATEGORY_LEARNT_PHRASES
+} from '../redux/constants'
```
Comment (andrzej-onja) [inline src/screens/Home.js]: those constants should be in translation file as its not related to redux

**Review by andrzej-onja — APPROVED** [review]

**Review by voromahery — APPROVED** [review]

**Review by Mickaellah — APPROVED** [review]

**Pull request merged**